lib/internal/crypto/certificate.js

```diff
@@ -6,20 +6,37 @@ const {
   certVerifySpkac
 } = process.binding('crypto');
 
+const errors = require('internal/errors');
+const { isArrayBufferView } = require('internal/util/types');
+
 const {
   toBuf
 } = require('internal/crypto/util');
 
-function verifySpkac(object) {
-  return certVerifySpkac(object);
+function verifySpkac(spkac) {
+  if (!isArrayBufferView(spkac)) {
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'spkac',
+                               ['Buffer', 'TypedArray', 'DataView']);
```

Comment:
Why only `spkac` doesn't have `string` in the error message?

Reply:
don't know. This was the way it was already. We can add string as an option in a separate follow on commit